src/Components/Favorites/Favorites.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Favorites.css";
+import FavoriteCard from "../FavoriteCard/FavoriteCard";
+
+const Favorites = ({ favorites, setFavorites }) => {
+
+  const favoriteBreweries = favorites.map(favorite => {
+    return (
+      <FavoriteCard 
+        id={favorite.id}
+        key={favorite.id}
+        name={favorite.name}
+        city={favorite.city}
+        state={favorite.state}
+        type={favorite.type}
+        isFavorite={favorite.isFavorite}
+        favorites={favorites}
+        setFavorites={setFavorites}
+      />
+    );
+  });
+
+  return (
+    <div className="favorites-container">
+      {favorites.length ? favoriteBreweries : <h3 className="no-favorites-message">You haven't added any favorites yet!</h3>}
```

Comment:
I have yet to be able to add error handling as I go so I know this means you have a better understanding of the concept than I do 👍